iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/IcebergRestUtils.java

```diff
@@ -71,4 +80,50 @@ public static Instant calculateNewTimestamp(Instant currentTimestamp, int hours)
     }
     return nextHourDateTime.atZone(ZoneId.systemDefault()).toInstant();
   }
+
+  public static String getCatalogName(String rawPrefix) {
+    String prefix = normalizePrefix(rawPrefix);
+    Preconditions.checkArgument(
+        !IcebergConstants.ICEBERG_REST_DEFAULT_CATALOG.equals(prefix),
+        String.format("%s is conflict with reserved key, please replace it", prefix));
```

Comment:
"...is conflicted with reserved catalog name"

Reply:
updated